### ISSUE TITLE
[BEST PRACTICE]: Add a DOCTYPE to prevent browsers entering quirks mode

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#5960]
## Description

A few pages are missing an HTML5 `<!DOCTYPE html>` declaration. This can cause browsers to kick down into Quirks mode and introduce bugs that might interfere with assistive technology or in ways we didn't anticipate.

## Type of Issue
Template

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* The Chrome dev tools Best Practices audit does not return a warning about missing DOCTYPE

## Environment

* Chrome latest, Incognito browser window
* MacOS Mojave
* https://www.va.gov/pittsburgh-health-care/operating-status/ (and possibly others)
